include/rtc/rtc.h

```diff
@@ -176,6 +199,113 @@ RTC_EXPORT int rtcDeleteTrack(int tr);
 
 RTC_EXPORT int rtcGetTrackDescription(int tr, char *buffer, int size);
 
+// Media
+#if RTC_ENABLE_MEDIA
+
+/// Add track
+/// @param pc Peer connection id
+/// @param codec Codec
+/// @param payloadType Payload type
+/// @param ssrc SSRC
+/// @param _mid MID (default: video|audio)
+/// @param _direction Direction (default RTC_SENDONLY)
+/// @param _name Name (optional)
+/// @param _msid MSID (optional)
+/// @returns Track id
+RTC_EXPORT int rtcAddTrackEx(int pc, rtcCodec codec, int payloadType, uint32_t ssrc, const char *_mid, rtcDirection * _direction, const char *_name, const char *_msid);
```

Comment:
I don't really like introducing a pointer for an optional value, it's confusing and rather cumbersome to call as you need to introduce local variables. `rtcDirection * _direction` should be `rtcDirection _direction` and the user could pass `RTC_DIRECTION_UNKNOWN` i.e; `0` to get the default behavior.